### DISCUSSION TITLE
AI 與生態環境

### DISCUSSION THREAD

**Discussion by JoannaLai909** [1101-GS4719-C/Python-Community-GS4719-C#2058] [General]
https://www.cw.com.tw/article/5130698
近期常看到許多新聞聊到關於AI對人類生活的影響，其中最讓我震驚的是生成式AI的排碳量，訓練一個模型的二氧化碳排放量竟然相當於美國汽車平均壽命使用下碳排放的5倍。在此之前我一直以為AI不過只是用電腦運作出來的東西而已，看到新聞我才知道原來AI在學習的過程中也會對環境造成影響，因此我現在對於AI的想法也開始改觀。雖然現在越來越多國家或企業都在追求發展強大的AI，以增強未來趨勢的競爭力，卻忽略了生態環境。我認為未來我們應做的不應只專注於科技進步，我們也應努力找出能平衡科技發展與生態環境維護的方法，而不是為了其一，犧牲另一。

**Comment by hunteryang0108** [comment]
其實不只是AI，包括所有的雲端運算中心也是如此。使用者往往忽略了雲端也是實際存在於地球上某個地方的，當地運算中心為此付出的土地、水以及電力資源其實是非常龐大的。在現今科技飛快發展的時代，關注生態環境確實是非常重要的議題。

**Comment by peresonacifer** [comment]
我之前曾查詢過如何訓練AI模型，簡單來說，因為在進行大規模的數據訓練時，會需要用到硬件的計算資源，並且訓練 AI 模型可能需要數小時、數天，甚至數週的時間，所以整個過程會消耗大規模的電量，過程中的排碳量真的分同小可，所以我們一定要在綠能方面多加投資，盡可能讓AI模型的訓練不要對生態產生負面影響。
